_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,111 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- [IdentityAttribute introduction]({_docs_integrate/identityattribute-introduction.md }) --->
+
+## Send Request
```

Comment:
Since you are referring to an "http request" and not an "enmeshed Request" (like described in the Request and Response Introduction), I'd recommend to explicitly say so here to prevent confusion.